presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -226,39 +226,49 @@ Property Name                                      Description
-                                                   URI is used by default, and the rest of the URIs are
-                                                   fallback metastores. This property is required.
-                                                   Example: ``thrift://192.0.2.3:9083`` or
-                                                   ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
+======================================================== ============================================================ ============
+Property Name                                            Description                                                  Default
+======================================================== ============================================================ ============
+``hive.metastore.uri``                                   The URI(s) of the Hive metastore to connect to using the
+                                                         Thrift protocol. If multiple URIs are provided, the first
+                                                         URI is used by default, and the rest of the URIs are
+                                                         fallback metastores. This property is required.
+                                                         Example: ``thrift://192.0.2.3:9083`` or
+                                                         ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
 
-``hive.metastore.username``                        The username Presto uses to access the Hive metastore.
+``hive.metastore.username``                              The username Presto uses to access the Hive metastore.
 
-``hive.metastore.authentication.type``             Hive metastore authentication type.
-                                                   Possible values are ``NONE`` or ``KERBEROS``
-                                                   (defaults to ``NONE``).
+``hive.metastore.authentication.type``                   Hive metastore authentication type.
+                                                         Possible values are ``NONE`` or ``KERBEROS``
+                                                         (defaults to ``NONE``).
 
-``hive.metastore.thrift.impersonation.enabled``    Enable Hive metastore end user impersonation.
+``hive.metastore.thrift.impersonation.enabled``          Enable Hive metastore end user impersonation.
 
-``hive.metastore.service.principal``               The Kerberos principal of the Hive metastore service.
+``hive.metastore.thrift.client.tls.enabled``             Whether TLS security is enabled.                             ``false``
 
-``hive.metastore.client.principal``                The Kerberos principal that Presto uses when connecting
-                                                   to the Hive metastore service.
+``hive.metastore.thrift.client.tls.keystore.path``       Path to the PEM or JKS key store.
```

Comment:
It's probably worth documenting whether this is expected to be an absolute or relative path (and then what it's relative to).  (FWIW, allowing a relative path is probably a security issue.)

Reply:
It can be either. Relative paths are often used for configuration, and this is safe because the working directory is guaranteed to be fixed by the launcher. It comes down to how you want to deploy and configure it.

Reply:
It can be either. Relative paths are often used for configuration, and this is safe because the working directory is guaranteed to be fixed by the launcher. It comes down to how you want to deploy and configure it.